AU/Public/Push-Package.ps1

```diff
@@ -14,7 +14,7 @@ function Push-Package() {
         [switch] $All
     )
     $api_key =  if (Test-Path api_key) { Get-Content api_key }
-                elseif (Test-Path ..\api_key) { Get-Content ..\api_key }
+                elseif (Test-Path (Join-Path '..' 'api_key')) { Get-Content (Join-Path '..' 'api_key') }
```

Comment:
Mhm.. this quickly became hard to look at...

Reply:
Yes, it really does. 

That could be resolved by making it into a variable.
`$apikeypath = (Join-Path '..' 'api_key')`

Reply:
Just ranting...